lib/new-admin/graphql/schema.js

```diff
@@ -190,6 +198,13 @@ const resolvers = {
     config: () => settingsLoader.getConfig()
   },
   Mutation: {
+    machineAction: (...[, { deviceId, action }]) => machineLoader.setMachine({ deviceId, action })
+      .then(() => machineLoader.getMachineNames())
+      .then(machines => machines.find(m => m.deviceId === deviceId))
+      .then(it => {
+        notify()
+        return it
+      }),
```

Comment:
We should try and keep with the same logic as before, throwing a error for machine not found. 

The logic should be placed on another file like we do with the other requests here. Feel free to create machinesjs file inside new-admin.

Reply:
moved to a new file and also I am using the USER_INPUT_ERROR that comes with apollo for the machine not found